spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -974,6 +974,28 @@ public void testPartitionedImportFromEmptyPartitionDoesNotThrow() {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void addDataParallelListing() {
```

Comment:
Nit: If we could follow the existing naming convention in the test class it helps readability. `testAddFilesWithParallelListing`